package.json

```diff
@@ -42,15 +41,17 @@
     "tweetnacl-util": "^0.15.0"
   },
   "devDependencies": {
-    "@metamask/eslint-config": "^2.0.0",
+    "@metamask/eslint-config": "^4.1.0",
     "@types/node": "^10.17.13",
-    "@typescript-eslint/eslint-plugin": "^2.20.0",
-    "@typescript-eslint/parser": "^2.20.0",
+    "@typescript-eslint/eslint-plugin": "^4.12.0",
+    "@typescript-eslint/parser": "^4.12.0",
     "browserify": "^16.5.1",
     "eslint": "^6.8.0",
-    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-json": "^2.1.2",
```

Comment:
We should consider dropping this in favour of `prettier`. That's what we did on the extension.

Reply:
I'd like to see us add `prettier` to our base `eslint` config first, and add the `prettier` plugin as a peerDependency. In fact, I was discussing this with @brad-decker earlier, and I also suggested that we do this: https://github.com/MetaMask/eslint-config/issues/104

Reply:
Oh I meant specifically for linting `.json` files. It was added to the extension to lint JSON before it was used for JavaScript.

But sure, we can wait. That works too.